ui/app/templates/components/dashboard/secrets-engines-card.hbs

```diff
@@ -0,0 +1,74 @@
+<Hds::Card::Container @hasBorder={{true}} class="has-padding-l secrets-engines-card">
+  <h3 class="title is-4 has-left-margin-xxs" data-test-dashboard-secrets-engines-header>Secrets engines</h3>
+  <Hds::Table @caption="Five secrets engines" class="is-border-spacing-revert" data-test-dashboard-secrets-engines-table>
+    <:body as |B|>
+      {{#each this.filteredSecretsEngines as |backend|}}
+        <B.Tr data-test-secrets-engines-row={{backend.id}}>
+          <B.Td class="is-flex-between is-flex-center has-gap-m">
+            <div>
+              <div class="is-flex-center">
+                {{#if backend.icon}}
+                  <ToolTip @horizontalPosition="left" as |T|>
+                    <T.Trigger>
+                      <Icon @name={{backend.icon}} class={{unless backend.isSupportedBackend "has-text-grey"}} />
+                    </T.Trigger>
+                    <T.Content @defaultClass="tool-tip">
+                      <div class="box">
+                        {{or backend.engineType backend.path}}
+                      </div>
+                    </T.Content>
+                  </ToolTip>
+                {{/if}}
+                {{#if backend.path}}
+                  {{#if backend.isSupportedBackend}}
```

Comment:
I think we talked about this at the meeting before and the decision was to show supported backends then unsupported ones! I will confirm with Yongqi!

Reply:
Just confirmed that we want to show both supported + unsupported backends since the goal is to make it as similar to the secrets engines page as possible.